### PR TITLE
DEV-30684 update Term API docs

### PR DESCRIPTION
**Pull request by ruskin-jano**
https://acrolinx.atlassian.net/browse/DEV-30684

### CODE REVIEW

File: term-api.md [hunk 1]
```diff
@@ -546,7 +546,6 @@ Otherwise, that is, no entry of the same UUID exists, a new entry will be create
 
 Supported media types are:
 
-- `application/vnd.acrolinx.actif+json`: ACTIF in JSON
```
Comment (robertgrabowski) [inline term-api.md:549]: @ruskin-jano There are three other endpoints which all claim to support "application/vnd.acrolinx.actif+json", which I'm pretty sure they don't either. Could you remove the three other occurrences of the "ACTIF in JSON" line as well from this document?

File: term-api.md [hunk 2]
```diff
@@ -566,7 +565,7 @@ Return codes:
 Example (sending [example-entry.actif.xml](doc/example-entry.actif.xml)):
 
 ```bash
-curl -i -H "Accept: application/json" -H "Content-Type: application/vnd.acrolinx.actif+xml" -H "Authorization: session ee7ac269aed25b4e" -X POST http://${serverHostName}:8031/iq/services/v7/rest/terminology/entries -d "`cat doc/example-entry.actif.xml`"
+curl -i -H "Accept: application/xml" -H "Content-Type: application/vnd.acrolinx.actif+xml" -H "Authorization: session ee7ac269aed25b4e" -X POST http://${serverHostName}:8031/iq/services/v7/rest/terminology/entries -d "`cat doc/example-entry.actif.xml`"
```
Comment (robertgrabowski) [inline term-api.md:568]: I suppose this change isn't necessary here - it controls the desired response format, and the response of this POST request will never contain content anyway (you only ever get a 201 or 204). Plus I don't know whether `Accept: application/xml` wouldn't actually break anything.

File: term-api.md (hunk 1 above)
Comment (ruskin-jano) [reply term-api.md:549]: Removed "application/vnd.acrolinx.actif+json" for the other endpoints as well

File: term-api.md (hunk 2 above)
Comment (ruskin-jano) [reply term-api.md]: Changed Accept: application/xml back to Accept: application/json

**Review by robertgrabowski — APPROVED** [review]
@ruskin-jano Thanks for making the changes!

**Pull request merged**